lib/elixir/lib/system.ex

```diff
@@ -499,4 +499,94 @@ defmodule System do
         raise ArgumentError, "invalid environment key-value #{inspect other}"
     end
   end
+
+  @doc """
+  Returns the current monotonic time in the `:native` time unit.
+
+  This time is monotonically increasing and starts in an unspecified point in
+  time. See
+  [`erlang:monotonic_time/0`](http://www.erlang.org/doc/man/erlang.html#monotonic_time-0).
+
+  For more information, see the [chapter on time and time
+  correction](http://www.erlang.org/doc/apps/erts/time_correction.html) in the
+  Erlang docs.
+
+  Inlined by the compiler into `:erlang.monotonic_time/0`.
+  """
+  @spec monotonic_time() :: integer
+  def monotonic_time do
+    :erlang.monotonic_time()
+  end
+
+  @doc """
+  Returns the current monotonic time in the given time unit.
+
+  See [`erlang:monotonic_time/1`](http://www.erlang.org/doc/man/erlang.html#monotonic_time-1).
+
+  For more information, see the [chapter on time and time
+  correction](http://www.erlang.org/doc/apps/erts/time_correction.html) in the
+  Erlang docs.
+
+  Inlined by the compiler into `:erlang.monotonic_time/1`.
+  """
+  @spec monotonic_time(:erlang.time_unit) :: integer
+  def monotonic_time(unit) do
+    :erlang.monotonic_time(unit)
+  end
+
+  @doc """
+  Returns the current system time in the `:native` time unit.
+
+  For more information, see the [chapter on time and time
+  correction](http://www.erlang.org/doc/apps/erts/time_correction.html) in the
+  Erlang docs.
+
+  Inlined by the compiler into `:erlang.system_time/0`.
+  """
+  @spec system_time() :: integer
+  def system_time do
```

Comment:
@fishcakez yep, `time` could work. Let's wait for @josevalim's seal of approval?